man/systemd.network.xml

```diff
@@ -2457,6 +2457,25 @@
         </listitem>
       </varlistentry>
 
+       <varlistentry>
+        <term><varname>CommonApplicationsKeptOverHead=</varname></term>
```

Comment:
hmm, the non-abbreviated form of "cake" sounds just weird, and overly hard to type, and it feels like it was specifically picked to have the nice sounding name "cake", hence I'd just use this abbreviated here, i.e. "CakeOverHeade=" and so on